web/webapp/src/main/java/no/nav/foreldrepenger/melding/web/app/exceptions/ConstraintViolationMapper.java

```diff
@@ -32,13 +29,13 @@ public Response toResponse(ConstraintViolationException exception) {
             String feltNavn = getFeltNavn(constraintViolation.getPropertyPath());
             feilene.add(new FeltFeilDto(feltNavn, constraintViolation.getMessage(), null));
         }
-        List<String> feltNavn = feilene.stream().map(felt -> felt.getNavn()).collect(Collectors.toList());
+        List<String> feltNavn = feilene.stream().map(FeltFeilDto::getNavn).collect(Collectors.toList());
 
-        Feil feil = FeltValideringFeil.FACTORY.feltverdiKanIkkeValideres(feltNavn);
-        feil.log(log);
+        var feil = FeltValideringFeil.feltverdiKanIkkeValideres(feltNavn);
+        log.warn(feil.getMessage());
```

Comment:
Her kan det nok logges direkte uten å opprette en FeltValideringFeil

Reply:
Ja, tar denne med i neste del!

Reply:
hmm...svarte litt fort. Her brukes samme melding for å logge en warning og til å legge inn en feilDto, så tenker det er greit å bruke feltvalideringsFeil her?